config/locales/en.yml

```diff
@@ -124,12 +124,10 @@ en:
       last_name: Last name
       ssn: Social Security Number
       dob: Date of birth
-      ccn: Credit Card Number
+      ccn: Last 8 Digits of your Credit Card Number
```

Comment:
I abbreviated 'social security number' as `ssn` and 'credit card number' as `ccn`.

I think I pulled both from at least one potential proofing vendor, so that's at least one data point.